site/gatsby-site/src/templates/cite.js

```diff
@@ -230,8 +230,16 @@ function CitePage(props) {
       addToast({
         message: (
           <Trans i18n={i18n}>
-            Please <Link to={localizePath({ path: '/login', language: locale })}>log in</Link> to
-            subscribe
+            Please{' '}
+            <Link
+              to={localizePath({
+                path: `/login?redirectTo=${props?.location?.pathname}`,
+                language: locale,
+              })}
+            >
+              log in
+            </Link>{' '}
+            to subscribe
```

Comment:
@clari182 Please go to https://deploy-preview-1713--staging-aiid.netlify.app/es/cite/421/ and click on `Notificarme de actualizaciones` and you will see the toast in English

Reply:
fixed